_posts/izayak/2016-03-29-regex-exercise-ry.md

```diff
@@ -0,0 +1,18 @@
+---
+author: izayak
+title: "Ruotong's Regex Exercise
+layout: post
+---
+
+<iframe src="https://trinket.io/embed/python/b9c50ef805" width="100%" height="600" frameborder="0" marginwidth="0" marginheight="0" allowfullscreen></iframe>
+
+  
+**Reflection**
+The first two exercises help us practice the methods discussed in the textbook which can be quite useful. The re.search() and re.findall() are two tools that are helpful when people search through files for certain contents. The third exercise help us review the materials of exercise 4 of last assignment and how to convert a list to a string. These exercises deepen our understanding of strings and lists and require a comprehension of these two main sturctures. 
+
+  
+*Question*
+I first tried to do this in Python 3 of Trinket, but it cannot be shared, why?
```

Comment:
You'll need to make one from trinket.io/python3 for the share button to be active.  Please replace this trinket with one made from there.